config/applications.json

```diff
@@ -727,6 +727,14 @@
 		"link": "https://desktop.github.com/",
 		"winget": "Git.Git;GitHub.GitHubDesktop"
 	},
+	"WPFInstallgitkrakenclient": {
+		"category": "Development",
+		"choco": "git;gitkraken",
```

Comment:
Is this supposed to just be "gitkraken"? i.e. no git;

Reply:
p.s. make sure you change the PR to merge into the test branch not main :)

Reply:
I was wondering about that... copied from other GIT clients... I can easily remove

Reply:
Removed `git;`

Reply:
Changed to merge to test branch